redis/commands/core.py

```diff
@@ -5016,8 +4998,8 @@ def hset(
         key: Optional[str] = None,
         value: Optional[str] = None,
```

Comment:
Due to the encoder class and its encode function:
https://github.com/redis/redis-py/blob/1cbb97ecd7a0bf87aec332a257b589919385d90d/redis/_parsers/encoders.py#L14
 inputs of values can be more than just string, I think something like the following would be more accurate:
`Union[str, float, int, bool, bytes, memoryview]`